### PR TITLE
OWNERS file audit and update

### PR DESCRIPTION
**Pull request by baude**
Update OWNERS file specifically around `approvers`.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,33 +1,14 @@
...
-  - saschagrunert
-  - TomSweeneyRedHat
-  - umohnani8
-  - vrothberg
 reviewers:
-  - ashley-cui
   - baude
-  - cevich
   - edsantiago
   - flouthoc
```
Comment (mheon) [inline OWNERS:10]: On reviewers but not approvers?

File: OWNERS [hunk 2]
```diff
@@ -1,33 +1,14 @@
...
-  - giuseppe
-  - jwhonce
   - lsm5
   - Luap99
   - mheon
-  - mtrmac
   - rhatdan
```
Comment (Luap99) [inline OWNERS:14]: Dan isn't look at any PRs here

**Review by Luap99 — APPROVED** [review]
LGTM

**Review by lsm5 — APPROVED** [review]
LGTM apart from @mheon's comment.

**Pull request merged**